src/Sulu/Component/Content/Metadata/Factory/StructureMetadataFactory.php

```diff
@@ -52,6 +52,11 @@ class StructureMetadataFactory implements StructureMetadataFactoryInterface
      */
     private $cache = [];
 
+    /**
+     * @var \Doctrine\Inflector\Inflector
```

Comment:
add here `use \Doctrine\Inflector\Inflector;` for this:

```suggestion
     * @var Inflector
```